deploy/crds/certman_v1alpha1_certificaterequest_crd.yaml

```diff
@@ -95,12 +95,16 @@ spec:
               description: WebConsoleURL is the URL for the cluster's web console
                 UI.
               type: string
+            region:
```

Comment:
This does not lgtm, has a aws specific config in the top level doesn't make sense to all the other cloud provider, we may consider add the region info as an annotation in the CR object, or add another struct for platform specific, something like:

```
platform:
    aws:
      region: <region_name>
      <other platform related>
```

Reply:
That makes sense @wanghaoran1988 
I'd prefer to keep it as a property on the CRD, I see there is an existing `platformSecrets` property which could be updated to just `platform` and could contain the region info too
e.g.

```.yaml
platform:
  aws:
    credentials:
    region:


platform:
  gcp:
    credentials:
```

wdyt?

Reply:
Remove the duplicate "platform" ?
```
platform:
  aws:
    credentials:
    region:
  gcp:
    credentials:
```

Reply:
@wanghaoran1988 Yes, my mistake. Exactly that